packages/app-file-manager/src/admin/graphql.ts

```diff
@@ -0,0 +1,29 @@
+import gql from "graphql-tag";
```

Comment:
Functionally everything is good of course. But just for consistency sakes, please rename query/mutation into something like `GET_SETTINGS` / `UPDATE_SETTINGS`. 

We're basically trying to mimic the same approach that's used on [Apollo docs](https://www.apollographql.com/docs/react/data/queries/).